### DISCUSSION TITLE
How to enable nodejs profiles?

### DISCUSSION THREAD

**Discussion by gcb** [netblue30/firejail#6535] [Q&A] [answered]
I see my distro (arch) have all the nodejs/node/npm profiles. But the main firecfg.config doesn't mention any of them and so node is running free.

what's the correct way to enable it? i tried just adding "node" there since all the profiles mention they all use `/bin/env node` but that didn't seem to be the correct answer.

Relates to:

* #6537

**Comment by gcb** [comment]
hum... apparently just adding "node" to `firecfg.config` worked, but for some reason firejail doesn't show the usual sandbox info when starting node. alas `firejail --list` do show node is running in a sandbox.

**Comment by kmk3** [comment, marked as answer]
> hum... apparently just adding "node" to `firecfg.config` worked,

Yes, for each "foo.profile", add "foo" to firecfg.config.

Note: Each profile that is commented in firecfg.config should be generally
treated as experimental and/or as not working properly in a common setup.

> but for some reason firejail doesn't show the usual sandbox info when
> starting node. alas `firejail --list` do show node is running in a sandbox.

This is due to `quiet`.

All/most CLI programs use `quiet` to avoid interfering with the program output
and with anything that might use it.

**Reply by rusty-snake** [reply]
> experimental and/or as not working properly in a common setup.

For CLI program this usually means that they work with firejail for some but not all working directories in an interactive shell but is known to break common shell scripts.

**Reply by rusty-snake** [reply]
> > alas firejail --list do show node is running in a sandbox.

> This is due to quiet.

--list should not be affect by quiet.
